students/anbaopham/lesson08/circle.py

```diff
@@ -0,0 +1,75 @@
+import math
+class Circle(object):
+    def __init__(self, radius):
+        self.radius = radius
+        #self.diameter = radius * 2.0
+    # @property
+    # def radius(self):
+    #     return self.diameter / 2.0
+    # @radius.setter
+    # def radius(self, radius):
+    #     self.diameter = radius * 2.0
+
+    @property
+    def diameter(self):
+        return self.radius * 2
+    @diameter.setter
+    def diameter(self, diameter):
+        self.radius = diameter / 2
+
+    def area(self):
+        return math.pi * self.radius ** 2.0
+
+    @classmethod
+    def from_diameter(cls, diameter):
+        radius = diameter / 2
+        return cls(radius)
+
+    def __str__(self):
+        return "Circle with radius: {}".format(self.radius)
+
+    def __repr__(self):
+        return "Circle({})".format(self.radius)
+
+    def __add__(self, other):
+        self.radius += other.radius
+        return Circle(self.radius)
```

Comment:
Need a blank line here.  Always a blank line between functions, everywhere in the code